app-misc/sl/sl-5.02.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit toolchain-funcs
+
+DESCRIPTION="sophisticated graphical program which corrects your miss typing"
+HOMEPAGE="http://www.tkl.iis.u-tokyo.ac.jp/~toyoda/index_e.html https://github.com/mtoyoda/sl/"
+SRC_URI="https://github.com/mtoyoda/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="freedist"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~hppa ~ppc ~ppc64 ~sparc ~x86 ~x86-fbsd ~x86-interix ~amd64-linux ~x86-linux ~ppc-macos"
+IUSE="l10n_ja"
+
+RDEPEND="sys-libs/ncurses:0="
+DEPEND="${RDEPEND}"
+
+src_prepare() {
+	default
+	sed -e "s/-lncurses/$($(tc-getPKG_CONFIG) --libs ncurses)/" -i Makefile
+}
+
+src_install() {
+	dobin sl
+	doman ${PN}.1
+	local DOCS=( README.md )
+	if use l10n_ja; then
```

Comment:
Honestly, I would just install them unconditionally instead of bothering people with USE flags for two small files. But it's your choice.